### ISSUE TITLE
Naming the RV32E ABI with -mabi

### ISSUE THREAD

**Issue by asb** [lowRISC/riscv-toolchain-conventions#3]
Should the RV32E ABI be explicitly targeted using `-mabi=ilp32e`? This would seem consistent with the other supported ABIs having unique `-mabi` strings. Of course for RV32E, no other ABI string is valid.

**Comment by asb** [comment]
@palmer-dabbelt, @kito-cheng - thoughts? -mabi=ilp32e makes a lot of sense to me.

**Comment by kito-cheng** [comment]
Yeah, sounds good idea, I'll take the gcc and binutils part.

**Comment by palmer-dabbelt** [comment]
I agree that we need some way for `-march=rv32i` to target the E ABI.  `-march=rv32i -mabi=ilp32e` seems good to me.
